en/firefox/features/fingerprinting.ftl

```diff
@@ -0,0 +1,39 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/features/fingerprinting/
```

Comment:
Oops, just realized this URL changed to https://www-dev.allizom.org/firefox/features/block-fingerprinting/. So that's at least one fix we need to do.

Reply:
Yes, I was checking on this and the link didn't work. I thought it was not  set up on the staging yet. I assume I can view all the pages on staging for this PR. The links are helpful with the context.